packages/easy-coding-standard/tests/Set/Array/Fixture/doctrine_config.php.inc

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->extension('doctrine', ['dbal' => ['url' => '%env(resolve:DATABASE_URL)%', 'charset' => 'utf8mb4', 'default_table_options' => ['collate' => 'utf8mb4_unicode_ci']], 'orm' => ['auto_generate_proxy_classes' => true, 'naming_strategy' => 'doctrine.orm.naming_strategy.underscore_number_aware', 'auto_mapping' => true, 'mappings' => ['App' => ['is_bundle' => false, 'type' => 'annotation', 'dir' => '%kernel.project_dir%/src/Entity', 'prefix' => 'App\Entity', 'alias' => 'App']]]]);
+};
+
+?>
+-----
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->extension('doctrine', [
+        'dbal' => [
+            'url' => '%env(resolve:DATABASE_URL)%',
+            'charset' => 'utf8mb4',
+            'default_table_options' => [
+                'collate' => 'utf8mb4_unicode_ci',
+            ],
+        ],
+        'orm' => [
+            'auto_generate_proxy_classes' => true,
+            'naming_strategy' => 'doctrine.orm.naming_strategy.underscore_number_aware',
+            'auto_mapping' => true,
+            'mappings' => [
+                'App' => [
+                    'is_bundle' => false,
+                    'type' => 'annotation',
+                    'dir' => '%kernel.project_dir%/src/Entity',
+                    'prefix' => 'App\Entity',
+                    'alias' => 'App',
+                ],
+            ],
+        ],
+    ]);
```

Comment:
If you could cut it down to 3 lines, that cause the issue, it would really make the debugging fixer much easier.

The fixer stop on every space/newline/arrow token, so before reaching the broken one, there is already dozens of unrelated invokes that load to cognition.

Reply:
simplified it to 2 cases.